frontend/src/metabase/metadata/pages/DataModel/components/FieldSection/BehaviorSection/RemappingPicker/RemappingPicker.tsx

```diff
@@ -154,6 +155,8 @@ export const RemappingPicker = ({
     setHasChanged(false);
     setIsChoosingInitialFkTarget(false);
 
+    trackMetadataChange("display_values");
```

Comment:
We need this call in `handleCustomMappingChange` and `handleFkRemappingFieldChange` as well